frontend/package.json

```diff
@@ -9,6 +9,7 @@
     "release-docker": "export NODE_OPTIONS=--max_old_space_size=8000 && vite build --mode release",
     "lint": "eslint --cache --ext .js,.ts,.tsx,.vue src",
     "lint:fix": "eslint --cache --ext .js,.ts,.tsx,.vue --fix src",
+    "type-check": "vue-tsc --noEmit --skipLibCheck",
```

Comment:
How about adding `type-check` to GitHub actions

Reply:
Negative for now I think. It's crazily slow and not a blocker.